examples/joystick/Joystick.cpp

```diff
@@ -0,0 +1,176 @@
+
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/Graphics.hpp>
+#include <sstream>
+#include <iomanip>
+#include <string>
+#include <map>
+
+
+namespace
+{
+    typedef std::map<std::string, std::pair<sf::Text, sf::Text> > Texts;
```

Comment:
While convenient and using existing classes, I'd really just define a small struct for the values here, even if it's just to identify the components as `label` and `value` or something like that. It's about readability and the chained expressions with `first` or `second` just feel a bit odd.

Reply:
Agreed. `std::pair` is never a good choice.